supervisor/utils/logging.py

```diff
@@ -59,6 +61,52 @@ def close(self) -> None:
         self.listener = None
 
 
+class HAOSLogHandler(JournaldLogHandler):
```

Comment:
If the Core should log through Systemd as well, I think it would be good to extract this to a shared Python package.

Reply:
Yeah make sense, it is fine by me to have this in Supervisor for the time being.